packages/lambda-powertools-cloudwatchevents-client/package.json

```
@@ -21,4 +22,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
+}
```

Nitpicky I know but could we get the newlines re-added? (a few repeated examples in this PR)
```suggestion
}

```

---

All done!